src/main/java/tira/TiraException.java

```diff
@@ -0,0 +1,9 @@
+package tira;
+
+import java.lang.Exception;
+public class TiraException extends Exception{
+    public TiraException(String message) {
+        super(message);
+    }
+    // will revise the exceptions later
```

Comment:
It's nice to know that the exception classes will be later revised.
Keep up the good work! :+1: